docs/en/getting-started/for_local.md

```diff
@@ -149,27 +149,27 @@ juicefs format --storage s3 \
     sqlite3://myjfs.db myjfs
 ```
 
-In the above command, the database and file system names remain the same and the following information related to object storage is added.
+In the above command, the database and file system names remain the same, but the following information related to object storage needs to be added.
 
 - `--storage`: Used to set the storage type, e.g. oss, s3, etc.
-- `--bucket`: Used to set the Endpoint address of the object store.
+- `--bucket`: Used to set the Endpoint address of the object storage.
 - `--access-key`: Used to set the Object Storage Access Key ID.
 - `--secret-key`: Used to set the Object Storage Access Key Secret.
 
 :::note
 Please replace the information in the above command with your own object storage information.
 :::
 
-Once created, you can mount it.
+You can mount the file system once it is created.
 
 ```shell
 juicefs mount sqlite3://myjfs.db mnt
 ```
 
-As you can see, the mount command is exactly the same as when using local storage, because JuiceFS has already written the information about the object storage to the `myjfs.db` database, so there is no need to provide it again when mounting.
+As you can see, the mount command is exactly the same as using the local storage because JuiceFS has already written the information about the object storage to the `myjfs.db` database, so there is no need to provide it again when mounting.
```

Comment:
```suggestion
As you can see, the mount command is exactly the same as using the local storage because JuiceFS has already written the metadata about the object storage to the `myjfs.db` database, so there is no need to provide it again when mounting.
```